### PR TITLE
Added initial version of English translations

### PR DESCRIPTION
**Pull request by boerge1**

### CODE REVIEW

File: audio_messages_en.txt [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+mp3/0971_modifier_SleepTimer.mp3|Snooze mode
+mp3/0972_modifier_FreezeDance.mp3|Stop Dance - TonUINO plays stop dance with you and randomly stops playback briefly for you.
+mp3/0973_modifier_Locked.mp3|Lock TonUINO
+mp3/0974_modifier_Toddler.mp3|Crawler mode - All keys of the TonUINO are locked for the very small ones. Cards still work.
+mp3/0975_modifier_KinderGarden.mp3|Kindergarten mode - In order to avoid conflicts, new cards are not played immediately, but only after the current song is over. The forward and back buttons are locked.
+mp3/0976_modifier_repeat1.mp3|Repeat title - repeat the current title endlessly.
+mp3/0980_admin_lock_intro.mp3|Please select if and how the admin menu should be protected.
+mp3/0981_admin_lock_disabled.mp3|No protection - the admin menu can be activated at any time by pressing all three buttons.
+mp3/0982_admin_lock_card.mp3|Admin card only - the admin menu can only be opened with an admin card. A new admin card can be learned at any time.
+mp3/0983_admin_lock_pin.mp3|Codeeingabe - das Adminmenü wird durch eine vierstellige Pin geschützt. Wenn du diese Option auswählst musst du als nächstes die Pin festlegen.
```
Comment (mweimerskirch) [inline audio_messages_en.txt:84]: One translation missing here

File: audio_messages_en.txt [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+mp3/0961_timer_5.mp3|5 minutes.
+mp3/0962_timer_15.mp3|15 minutes.
+mp3/0963_timer_30.mp3|30 minutes.
+mp3/0964_timer_60.mp3|60 minutes.
+mp3/0965_timer_disabled.mp3|do not switch off automatically
+mp3/0970_modifier_Intro.mp3|Now please select your modification card with the volume keys.
+mp3/0971_modifier_SleepTimer.mp3|Snooze mode
+mp3/0972_modifier_FreezeDance.mp3|Stop Dance - TonUINO plays stop dance with you and randomly stops playback briefly for you.
+mp3/0973_modifier_Locked.mp3|Lock TonUINO
+mp3/0974_modifier_Toddler.mp3|Crawler mode - All keys of the TonUINO are locked for the very small ones. Cards still work.
```
Comment (mweimerskirch) [inline audio_messages_en.txt:78]: I think this should be called "Toddler mode" (like the filename suggests).

File: audio_messages_en.txt [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+mp3/0912_admin_lock.mp3|Secure the admin menu.
+mp3/0913_pause_on_card_removed.mp3|Pause when card is removed.
+mp3/0919_continue_admin.mp3|and continuing in the admin menu
+mp3/0920_eq_intro.mp3|Please select a setting for the EQ with the volume keys and confirm it with the pause key.
+mp3/0921_normal.mp3|Normal
+mp3/0922_pop.mp3|Pop
+mp3/0923_rock.mp3|Rock
+mp3/0924_jazz.mp3|Jazz
+mp3/0925_classic.mp3|Classic
+mp3/0926_bass.mp3|Base
```
Comment (mweimerskirch) [inline audio_messages_en.txt:55]: Bass

File: audio_messages_en.txt (hunk 1 above)
Comment (boerge1) [reply audio_messages_en.txt:84]: Translation done.

File: audio_messages_en.txt (hunk 2 above)
Comment (boerge1) [reply audio_messages_en.txt:78]: Changed

File: audio_messages_en.txt (hunk 3 above)
Comment (boerge1) [reply audio_messages_en.txt]: Changed

**Comment by boerge1** [comment]
@mweimerskirch I did all proposed changes. Thanks for reviewing!

**Pull request merged**